source/clear-linux/reference/collaboration/documentation/simple.rst

```diff
@@ -0,0 +1,302 @@
+and appropriate to the audience's reading skills and knowledge.
+
+Simple English improves the clarity of procedural technical writing,
+makes translation easier, and improves comprehension for people whose
+first language is not English.
+
+|CLOSIA| does not use controlled language, which restricts the writer's
+vocabulary to a list of approved words. Some preferences are evidently in
+place.
+
+Short sentences and paragraphs
+******************************
+
+Clear writing should average 15 to 20 words per sentence. This does not
+mean every sentence should be the same length. Vary your writing by
+mixing short sentences with longer ones, but stick to the basic
+principle of one main idea in a sentence, plus one additional point if
+needed.
+
+Similarly, restrict your paragraph length to about six sentences.
+Remember the basic structure of a paragraph: Introduction, body and
+conclusion. Both the introduction and the conclusion should be one
+sentence long. The body of a paragraph should never exceed four
+sentences. Here less is more.
+
+Simple words
+************
+
+Choosing simple words increases reader comprehension and reduces
+ambiguity. Here are some guidelines on making good simple word choices:
+
+* Avoid jargon. Jargon is a type of language that is only understood
+  by a particular group of people, such as an industry or a club. You
+  can use jargon when writing for an audience who will understand, but
+  avoid over using it, especially on the general public.
+* Be consistent. Use one term for each concept or action and use it
+  consistently. Don't use a different term for the same object or
+  action when you refer to it subsequently.
+* Keep your style plain but avoid dullness. Avoid clichés, idioms, and
+  metaphors. Many of these devices are not easily understood across
+  different cultures and can lead to confusion.
+* Avoid "fancy" words and phrases. The goal is to get the information
+  across, not to impress the reader with your vocabulary, so avoid
+  bureaucratic, flowery or literary style. Here are some examples of
+  "formal" words to avoid and preferred "informal" alternatives in
+  parentheses:
+
+*  commence (start, begin)
+*  consequently (so)
+*  in excess of (more than)
+*  in the event of (if)
+*  prior to (before)
+*  should you wish (if you want)
+*  utilize (use)
+*  instance (example)
+
+Strong verbs
+************
+The stronger and clearer you can make your verbs, the more directly you
+communicate information to your audience.
+
+Keep these basic guidelines in mind as you check your verbs:
+
+* Use imperatives.
+* Use active voice not passive voice.
+* Avoid linking verbs; is, seems, becomes.
+* Convert weak verbs and nominalizations to strong verbs.
+* Be concise.
+* Avoid "there are" and "it is" constructions. .. note:: The following
+  examples offer two versions of the same information. The incorrect
+  version always comes first and is formatted *in italics*. The correct
```

Comment:
I'm not sure what this sentence is telling me.

Reply:
fixed